events/world-moveit-day-2018/index.markdown

```diff
@@ -0,0 +1,102 @@
+layout: page
+slug: world_moveit_day_2018
+title: World MoveIt! Day 2018
+---
+
+<img src="/assets/images/wmd18/world_moveit_day_2018.png" alt="World MoveIt! Day" style="width: 700px;"/>
+
+## Date
+
+  * **Thursday, October 25th, 2018**
+  * See specific locations for details on times. 
+  * Note that the Tokyo and Singapore locations will have their events on Friday the 26th
+
+## Description
+
+World MoveIt! Day is an international hackathon to improve the MoveIt! code base, documentation, and community. We hope to close as many pull requests and issues as possible and explore new areas of features and improvements for the now seven year old framework. Everyone is welcome to participate from their local workplace, simply by working on open issues. In addition, a number of companies and groups host meetings on their sites all over the world. A video feed will unite the various locations and enable more collaboration. Maintainers will take part in some of these locations.
+
+## Locations
+
+- [Iron Ox](http://ironox.com), San Francisco, USA
+  - Organizer: Jon Binnery
+  - [Event details](https://www.eventbrite.com/e/world-moveit-day-2018-bay-area-tickets-49609912584)
+- [PickNik Consulting](http://picknik.ai), Boulder, USA
+  - Organizer: Dave Coleman
+- [Fraunhofer IPA](https://www.ipa.fraunhofer.de/en.html) / [RIC-EU](https://rosindustrial.org/ric-eu/), Stuttgart, Germany
+  - Organizer: Michael 'v4hn' Görner
+- [ROS-Industrial Asian Pacific Consortium](http://rosindustrial.org/ric-apac/), Singapore
+  - October 26
+  - Organizer: Chan Min Ling
+- [Tokyo Opensource Robotics Kyokai Association (TORK)](http://opensource-robotics.tokyo.jp/), Tokyo, Japan
+  - October 26
+  - Organizer: Ryosuke Tajima
+- [Southwest Research Institute](https://www.swri.org/), San Antonio, USA
+  - Organizer: Matt Robinson
+
+## General Information Contacts
+
+  * Dave Coleman, Nathan Brooks, Rob Coleman // PickNik Consulting
+
+## Signup
+
+Please state your intent to join the event [on this form](https://docs.google.com/forms/d/e/1FAIpQLSdk_xMOVdqusdvT6vr5s6AnXV-GdtchCjrX-BXpyw642ahcBg/viewform). Note that specific locations will have their own signups in addition to this form.
+
+If you aren't near an organized event we encourage you to have your own event in your lab/organization/company and video conference in to all the other events. We would also like to mail your team or event some MoveIt! stickers to schwag out your robots!
+
+## Logistics
+
+### What version of MoveIt! should you use?
+
+We recommend the Kinetic LTS branch/release. The Melodic release is also a good choice but is new and has been tested less. The Indigo branch is considered stable and frozen - and only critical bug fixes will be backported.
+
+For your convenience, a VirtualBox image for ROS Kinetic on Ubuntu 16.04 is available [here](http://backend.picknik.ai/).
+
+### Finding Where You Can Help
+
+Suggested areas for improvement are tracked on MoveIt's GitHub repo via several labels:
```

Comment:
This may change, I think switching to a project board would be ideal, but in the interest of getting this page updated I think we should leave it as-is for now.

Reply:
Make that a short topic for the next maintainer meeting then.
Personally I do not share your conviction there.